holoviews/element/chart.py

```diff
@@ -340,3 +319,20 @@ def stack(cls, areas):
                                     new_type=Area)
             baseline += y
         return stacked
+
+
+class BoxWhisker(Chart):
+    """
+    BoxWhisker represent data as a distributions highlighting the
+    median, mean and various percentiles. It may have a single value
+    dimension and any number of key dimensions declaring the grouping
+    of each violin.
+    """
```

Comment:
I assume there a reason ``BoxWhisker`` can't also go into ``element/stats.py``?

Reply:
Yes, I initially moved it there, but it breaks unpickling of our test data and I can't move it and import it into charts because that would be circular.

Reply:
Ok. Should this be marked as something to fix in version 2.0 when we can break backwards compatibility?

Reply:
Yes, probably.

Reply:
Should I just file an issue once this is merged?